src/plugins/prebuilt-library/data-fetch/get-async-data.js

```diff
@@ -419,6 +419,30 @@ export function getAsyncData() {
 		}
 	}
 
+	/**
+	 * Send Event to Backend.
+	 *
+	 * @param {string} event_label
+	 * @param {object} event_data
+	 *
+	 * @return {Promise<object>} Promise returns object
+	 */
+	async function sendEvent( event_label, event_data ) {
+		try {
+			return await apiFetch( {
+				path: '/kb-design-library/v1/handle_event',
+				method: 'POST',
+				data: {
+					event_label: event_label,
+					event_data: event_data ? event_data : '',
```

Comment:
This has to be an object, right?

```suggestion
					event_data: event_data ? event_data : {},
```